crates/sui-framework-snapshot/src/lib.rs

```diff
@@ -128,3 +127,33 @@ pub fn load_bytecode_snapshot(protocol_version: u64) -> anyhow::Result<Vec<Syste
 pub fn manifest_path() -> PathBuf {
     PathBuf::from(env!("CARGO_MANIFEST_DIR")).join("manifest.json")
 }
+
+/// Given a protocol version returns:
```

Comment:
```suggestion
/// Given a protocol version:
```
Could just remove "is returned" from L135 but suggest this approach